.changeset/gorgeous-cheetahs-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
+---
+
+Removed the `Label` component from the public exports. Use the `label` prop on the form components instead. When building a custom form component, use a `label` element with the `typography('two')` style mixin (don't forget about the `for` prop).
```

Comment:
```suggestion
Removed the `Label` component from the public exports. Use the `label` prop on the form components instead. When building a custom form component, use a `label` element with the `typography('two')` style mixin (don't forget about the `htmlFor` prop).
```

Reply:
Hm, I'd be inclined to stick to the vanilla `for` over the React-specific `htmlFor`. React devs should know that they need to translate it.

Reply:
Fine by me 👌 this is why it's approved. But since we only support React for now we could also make it explicit